addons/l10n_sa_edi_pos/static/src/js/PaymentScreen.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('l10n_sa_edi_pos.PaymentScreen', function(require) {
+    "use strict";
+
+    const PaymentScreen = require('point_of_sale.PaymentScreen');
+    const Registries = require('point_of_sale.Registries');
+
+    const { onMounted } = owl.hooks;
+
+    const PosSAPaymentScreen = PaymentScreen => class extends PaymentScreen {
+        //@Override
+        toggleIsToInvoice() {
+            // If the company is Saudi, POS orders should always be Invoiced
```

Comment:
and this makes the button disappear?

Reply:
it disables the toggling of the button so that the user doesn't toggle it off by mistake.